pyomo/contrib/gdpopt/util.py

```diff
@@ -253,6 +253,8 @@ def copy_var_list_values(from_list, to_list, config,
     Rounds to Binary/Integer if necessary
     Sets to zero for NonNegativeReals if necessary
     """
+    if len(from_list) != len(to_list):
+        raise ValueError('The lengths of from_list and to_list do not match.')
```

Comment:
Can we come up with a simple test for this line? Codecov is crying about it and it refers to the commented tests above.